docs/user/custom-resources/apirule/04-70-changes-in-apirule-v2.md

```diff
@@ -43,6 +43,14 @@ Versions `v2` and `v2alpha1` do not apply these default values. If the **corsPol
 
 **Required action**: To use default CORS values defined in the APIRule `v1beta1`, you must explicitly define them in the **corsPolicy** field.
 
+The following snippet shows an example of the default CORS configuration in the APIRule CR:
```

Comment:
I already added this default configuration to the paragraph https://github.com/kyma-project/api-gateway/pull/1751/files#diff-860f0ee1b91aef645e544022874ff7a4a20ddb65624b6d7318bbb4f682fcaabaR35
Could you replace it with your code snippet?